docs/astro/source/plot_pwn_evolution.py

```diff
@@ -17,4 +17,4 @@
 plt.ylabel('radius [pc]')
 plt.legend(loc=4)
 plt.loglog()
-plt.show()
+plt.savefig('test.png')
```

Comment:
Why is there a `savefig` call here?
Does this work with the Sphinx build, i.e. get shown in the HTML output.

Otherwise, @adonath, everything looks good ... merge?